bindings/gumjs/gumv8value.cpp

```diff
@@ -1146,7 +1146,22 @@ _gum_v8_native_pointer_parse (Handle<Value> value,
     *ptr = GSIZE_TO_POINTER (value.As<Number> ()->Value ());
     return TRUE;
   }
+  else
+  {
+    auto uint64 (Local<FunctionTemplate>::New (core->isolate, *core->uint64));
+    if (uint64->HasInstance (value)) 
+    {
+      *ptr = GSIZE_TO_POINTER(_gum_v8_uint64_get_value (value.As<Object> ()));
+      return TRUE;
+    }
 
+    auto int64 (Local<FunctionTemplate>::New (core->isolate, *core->int64));
+    if (int64->HasInstance (value)) 
+    {
+      *ptr = GSIZE_TO_POINTER(_gum_v8_int64_get_value (value.As<Object> ()));
+      return TRUE;
+    }
```

Comment:
These two should be swapped - we should check them in the same order as their chronological listing: signed then unsigned.